README.md

```diff
@@ -165,6 +170,7 @@ grunt.initConfig({
     default: {
       options: {
           path: 'test/js/*.html',
+          cwd: 'target/combined-sources',
```

Comment:
Right, that makes sense, thanks. Would you like me to create a different test case for it, or only leave the default case?